src/@chakra-ui/gatsby-plugin/components/Tooltip.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  cssVar,
+  defineStyle,
+  defineStyleConfig,
+  getToken,
+} from "@chakra-ui/react"
+
+const $arrowBg = cssVar("popper-arrow-bg")
+
+const baseStyle = defineStyle((props) => {
+  const tableBoxShadow = getToken("colors", "tableBoxShadow")(props.theme)
```

Comment:
This will not work. It looks like you are trying to grab a token that resides in the old `theme.ts` file, but Chakra will never see this file.

Because this shadow might only be used for the tooltip, I would suggest simply hardcoding the value here. Per the Figma file, only one `boxShadow` value is going to be used in both light and dark mode.